lidarr/command.go

```diff
@@ -87,20 +87,20 @@ func (l *Lidarr) SendCommandContext(ctx context.Context, cmd *CommandRequest) (*
 }
 
 // GetCommandStatus returns the status of an already started command.
-func (s *Lidarr) GetCommandStatus(commandID int64) (*CommandResponse, error) {
-	return s.GetCommandStatusContext(context.Background(), commandID)
+func (l *Lidarr) GetCommandStatus(commandID int64) (*CommandResponse, error) {
+	return l.GetCommandStatusContext(context.Background(), commandID)
 }
 
 // GetCommandStatusContext returns the status of an already started command.
-func (s *Lidarr) GetCommandStatusContext(ctx context.Context, commandID int64) (*CommandResponse, error) {
+func (l *Lidarr) GetCommandStatusContext(ctx context.Context, commandID int64) (*CommandResponse, error) {
 	var output CommandResponse
 
 	if commandID == 0 {
 		return &output, nil
 	}
 
 	req := starr.Request{URI: path.Join(bpCommand, fmt.Sprint(commandID))}
-	if err := s.GetInto(ctx, req, &output); err != nil {
+	if err := l.GetInto(ctx, req, &output); err != nil {
 		return nil, fmt.Errorf("api.Post(%s): %w", &req, err)
```

Comment:
This should say `api.Get(%s)` ... now I'm going to look at the example I pointed you to....

Reply:
Yup Sonarr is wrong too. You're welcome to fix it too.

Reply:
I mean, good news is, it passes the tests. Bad news is, you found another problem.

Reply:
1 small fix, sonarr too if you want, and I'll merge this up. :)